### ISSUE TITLE
`filesAreValid` is not called

### ISSUE THREAD

**Issue by green-arrow** [funkensturm/ember-cli-file-picker#10]
If you follow the documentation and extend the component, specifying a `filesAreValid` function, it is never called. After looking into the code, it looks like the issue might be in the `handleFiles` function:

```javascript
if(this.filesAreValid === 'function')
```

The above does not evaluate to true. I think it should be this:

```javascript
if(typeof(this.filesAreValid) === 'function')
```

Am I doing something wrong?

**Comment by fsmanuel** [comment]
@green-arrow you are totally right. can you open a PR and add a test?

**Comment by fsmanuel** [comment]
@green-arrow i released `0.0.7` with the fix. thanks a lot!
